package.json

```diff
@@ -57,11 +57,13 @@
     "@babel/runtime": "^7.11.2",
     "@react-spring/three": "^9.3.1",
     "@use-gesture/react": "^10.2.0",
+    "camera-controls": "^1.37.6",
     "detect-gpu": "^5.0.5",
     "glsl-noise": "^0.0.0",
     "lodash.clamp": "^4.0.3",
     "lodash.omit": "^4.5.0",
     "lodash.pick": "^4.4.0",
+    "maath": "^0.5.1",
```

Comment:
I don't know why yarn has re-ordered the "maath" dep 🤷‍♂️

I can manually re-ordered it if you want

Reply:
No worries, it tries to alphabetically sort.